src/firmware-updater.js

```diff
@@ -56,6 +57,18 @@ export default class FirmwareUpdater {
       .pipe(takeUntil(this.updatingError));
   }
 
+  setToolVersion(version) {
+    this.toolVersion = version;
+    if (semverCompare(version, '1.1.2' < 0)) {
```

Comment:
I think that there is a typo here 
```suggestion
    if (semverCompare(version, '1.1.2') < 0) {
```